BusRoutes/Model/Route.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace BusRoutes.Model
+{
+	public class Route
+	{
+		int id;
+
+		[JsonProperty("id")]
+		public int Id {
+			get {
+				return id;
+			}
+			set {
+				id = value;
+			}
+		}
+
+		string shortName;
```

Comment:
Avoid omitting visibility modifiers, private members should be prefixed with `private`. 

Also, try to group members in the following manner: fields, constructor, properties, methods, this helps a new fellow engineer to properly assess all fields, constructors, etc in a given class